src/modules/pages/LoginPage/LoginForm.scss

```diff
@@ -0,0 +1,45 @@
+@import 'index.scss';
+
+$form-width: 530px;
+$form-height: 387px;
```

Comment:
Not really a fan of declaring heights like this, can you just get the height with the content and spacing given by the elements inside it?